test/suite/controller/lintController.test.ts

```diff
@@ -111,6 +111,13 @@ describe("lintController.ts", async () => {
       const data = {
         success: true,
         messages: [
+          {
+            type: "error",
+            code: "no file",
+            message: "i have no file!",
```

Comment:
Oh, that's a general message, right? This is totally possible, see: https://github.com/mozilla/addons-code-manager/blob/38c56a8a98ad7b55ccf598690e6b93d067c8adce/src/reducers/linter.tsx#L102-L105